src/Chunk.ts

```diff
@@ -331,11 +331,10 @@ export default class Chunk {
 		for (const module of entryModules) {
 			const requiredFacades: FacadeName[] = Array.from(
 				new Set(
-					module.chunkNames.filter(({ isUserDefined }) => isUserDefined).map(({ name }) => name)
-				),
-				name => ({
-					name
-				})
+					module.chunkNames
+						.filter(({ isUserDefined }) => isUserDefined)
+						.map(({ name }) => ({ name }))
```

Comment:
This isn't the same as it won't do name string deduping in the set.

Reply:
great catch, thank you!  too bad this wasn't caught by a test 😞  I reverted and added a comment.

Reply:
Actually I think it does not change anything at the moment, which is why there is no test. The only situation where you have entries here is if the user used an object as entry point. In that case, the `name` properties are the keys of that object, and object keys deduplicate themselves. It does not hurt either, though.